data/transactions/logic/assembler_test.go

```diff
@@ -745,11 +658,13 @@ func TestAssembleRejectNegJump(t *testing.T) {
 	t.Parallel()
 	text := `wat:
 int 1
-bnz wat`
+bnz wat
+int 2`
```

Comment:
why `int 2` here and below?

Reply:
I was making the test harder to pass.  In my initial implementation, a bug that occurred was that this particular error reported the last line *always*.  So, to make things harder, I added another line to ensure I was reporting the error on the bnz line, not just the last line.